geometry_msgs/PoseStamped Pose
float32 r
int8 taskFlag